roles/shared_handlers/handlers/main.yml

```diff
@@ -1,7 +1,26 @@
 # This role holds reusable handlers that can be included in multiple playbooks to keep things DRY
+- name: precompile assets
+  command: bash -lc "bundle exec rake assets:precompile:primary RAILS_GROUPS=assets RAILS_ENV={{ rails_env }}" chdir="{{ current_path }}"
+  become: yes
+  become_user: "{{ unicorn_user }}"
+  notify: precompile nondigest assets
+
+- name: precompile nondigest assets
+  command: bash -lc "bundle exec rake assets:precompile:nondigest RAILS_GROUPS=assets RAILS_ENV={{ rails_env }}" chdir="{{ current_path }}"
+  become: yes
+  become_user: "{{ unicorn_user }}"
+
 - name: restart unicorn
   service:
     name: unicorn_{{ app }}
     state: restarted
   become: yes
   become_user: root
+  notify: restart nginx
```

Comment:
I believe that nginx only needs to be restarted if the nginx config changes. Is that wrong?

Reply:
I was thinking that too! I left this in because it's how the current handlers work, but I agree that we can probably cut it.

Reply:
I would just create a separate issue. Agree. We almost never change nginx config so if we don't restart it we could already have zero-downtime deployments :tada:

Reply:
Removed! :fire: :smile:

Reply:
@sauloperez Don't we have zero-downtime deployments? What is currently causing downtime?